repair/repair.cc

```diff
@@ -491,8 +491,8 @@ float repair::task_manager_module::report_progress() {
     uint64_t nr_ranges_finished = 0;
     uint64_t nr_ranges_total = 0;
     for (auto& x : _repairs) {
-        auto it = _tasks.find(x.second);
-        if (it != _tasks.end()) {
+        auto it = get_all_tasks().find(x.second);
+        if (it != get_all_tasks().end()) {
```

Comment:
This change of using `get_all_tasks()` instead of `_tasks` should have been a separate commit, to reduce noise in this one.